tech/tech/domain/entities/users.py

```
@@ -0,0 +1,21 @@
+        self.cpf = cpf
+        self.email = email
+        self.created_at = datetime.now()
+        self.updated_at = datetime.now()
+
+    def update_password(self, new_password: str):
+        self.password = new_password
+        self.updated_at = datetime.now()
+
+    def update_email(self, new_email: str):
+        self.email = new_email
+        self.updated_at = datetime.now()
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)